src/screens/TopicPageScreen/index.js

```diff
@@ -87,7 +86,7 @@ const TopicPageScreen = (props) => {
     : dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT;
   const headerShowHeight =
     navigationHeight + dimen.size.TOPIC_FEED_HEADER_HEIGHT + topPosition + normalize(4);
-  const headerHideHeight = dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT2 + topPosition;
+  const headerHideHeight = dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT2;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The value of `headerHideHeight` has been changed. Make sure this change doesn't affect the layout or animation of the header.
```diff
- const headerHideHeight = dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT2 + topPosition;
+ const headerHideHeight = dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT2;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->